src/widgets/textlabel.h

```diff
@@ -2,7 +2,6 @@
 #define TEXTLABEL_H
 
 #include <QLabel>
-#include <QMouseEvent>
```

Comment:
Can't remember off the top of my head which version of Qt and on which platform. Maybe it will come up again or not, will see when I get to the next round of platform testing. Merged.